frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -240,21 +240,50 @@ export const getReportBugLink = (cv: ClusterVersionKind): { label: string; href:
     return null;
   }
 
-  // Show a Bugzilla link for prerelease versions and a support case link for supported versions.
   const { major, minor, prerelease } = parsed;
-  const bugzillaVersion = major === 4 && minor <= 3 ? `${major}.${minor}.0` : `${major}.${minor}`;
-  const environment = encodeURIComponent(`Version: ${version}
-Cluster ID: ${cv.spec.clusterID}
-Browser: ${window.navigator.userAgent}
-`);
+  let productName;
+  switch (window.SERVER_FLAGS.branding) {
+    case 'openshift':
+    case 'ocp':
+      productName = 'OpenShift%20Container%20Platform';
+      break;
+    case 'online':
+      productName = 'OpenShift%20Online';
+      break;
+    case 'dedicated':
+      productName = 'OpenShift%20Dedicated';
+      break;
+    case 'azure':
+      productName = 'OpenShift%20Managed%20(Azure)';
+      break;
+    default:
+      productName = 'OKD';
+  }
+  const bugLabel = i18next.t('public~Report bug to Red Hat');
+
+  // Show an OpenShift Container Platform Bugzilla link for OKD
+  // until the OKD product components are updated.
```

Comment:
This is assuming management does in fact want to resume usage of OKD in Bugzilla.